network-api/networkapi/buyersguide/templates/social.html

```diff
@@ -0,0 +1,19 @@
+{% if mobile %}
+  <div class="social d-flex flex-column">
+    <a class="social-button social-button-fb body-large">Facebook</a>
```

Comment:
we'll have to add `href` as (value-less) attribute to these apparently. Otherwise browsers don't treat them as links but as pre-HTML5 name anchors O_o!